plugins/nextcloud/NextcloudAddressBook.php

```diff
@@ -1,90 +1,129 @@
 <?php
+
 use RainLoop\Providers\AddressBook\Classes\Contact;
 
 class NextcloudAddressBook implements \RainLoop\Providers\AddressBook\AddressBookInterface
 {
 	use RainLoop\Providers\AddressBook\CardDAV;
-	private const URI = 'webmail';
 	private $contactsManager;
-	private $key;
 
-	function __construct()
+	private const SETTINGS_KEY = 'nextCloudAddressBookUri';
+
+	private string $defaultUri = 'webmail';
```

Comment:
const is better for this

Reply:
@akhil1508 this variable actually set at constructor.. (it is more like `final` variable than `const`... my java brain tells me that... not very sure how php handle this kind of case).

Reply:
@akhil1508 done